packages/bierzo-wallet/src/routes/terms/components/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { Block, Link, Paragraph, Section, Title } from "medulas-react-components";
+import * as React from "react";
+
+/**
+ * Workaround to avoid "Comment location overlaps with node location" error
+ * https://github.com/prettier/prettier/issues/2347
+ */
+
+const addLinkTo = (link: string): JSX.Element => <Link to={link}>{link}</Link>;
+
+export default (): JSX.Element => {
+  return (
+    <React.Fragment>
+      <Title>Terms</Title>
+      <Section>
+        <Paragraph>
+          The website {addLinkTo("https://wallet.hugnet.iov.one")} (hereinafter: the “Website”) is owned and
```

Comment:
Can we pull out the website name into the configuration? I think we don't need the link here (user is already on that page). Just a "website name"

Reply:
So what should I put here? Just remove the link?

Reply:
Jap, `wallet.hugnet.iov.one` is fine for now as long as it is easy to configure